src/Components/Publishing/Sections/ImageCollection.tsx

```diff
@@ -15,26 +15,29 @@ interface ImageCollectionProps {
   targetHeight?: number
   gutter?: number
   sectionLayout?: SectionLayout
+  articleLayout?: Layout
```

Comment:
This component represents a section called `image_collection`. It's a group of `Image` and `Artwork` components in a row.

For Newsfeed, we don't want the ability to expand images into a fullscreen viewer. We have the ability to turn that feature off on the `Image` and `Artwork` components individually but we need to pass down this `articleLayout` prop to make that work within an `ImageCollection`.